staging/src/kubevirt.io/api/instancetype/v1alpha2/types.go

```diff
@@ -75,6 +75,13 @@ type VirtualMachineClusterInstancetypeList struct {
 //
 // CPU and Memory are required attributes with both requiring that their Guest attribute is defined, ensuring a number of vCPUs and amount of RAM is always provided by each instancetype.
 type VirtualMachineInstancetypeSpec struct {
+	// NodeSelector is the same object type as contained in VirtualMachineInstanceSpec.
+	// +optional
+	NodeSelector map[string]string `json:"nodeSelector,omitempty"`
+
+	// SchedulerName is the name of the custom K8s scheduler for the VMI Preset.
+	// +optional
+	SchedulerName string `json:"schedulerName,omitempty"`
```

Comment:
IMHO both of these are preferences, something that a user can overwrite in a VirtualMachine with their own choices etc, thoughts?

Reply:
Hm, hard to decide. In my opinion if someone creates Instancetype and they specify the `schedulerName` it should be always the same, same with `nodeSelector`. And it feel like it should always be enforced. But I don't have very strong opinion about this.

Reply:
Apologies for the delay getting back to this, yeah I understand. Instance types are currently used to ecapsulate only the resource and performance related attributes of a VMI but thinking about it we could extend that to include these schedulable attributes as well.

@davidvossel, @fabiand, any thoughts on this?

Reply:
@lyarwood I have dropped VMIPreset from this MR. Could you take another look?